src/test/java/org/broadinstitute/consent/pact/sam/SamPactTests.java

```diff
@@ -0,0 +1,207 @@
+package org.broadinstitute.consent.pact.sam;
+
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertNotNull;
+
+import au.com.dius.pact.consumer.MockServer;
+import au.com.dius.pact.consumer.dsl.PactDslWithProvider;
+import au.com.dius.pact.consumer.junit.PactProviderRule;
+import au.com.dius.pact.consumer.junit.PactVerification;
+import au.com.dius.pact.core.model.RequestResponsePact;
+import au.com.dius.pact.core.model.annotations.Pact;
+import com.google.api.client.http.HttpStatusCodes;
+import java.util.List;
+import java.util.Map;
+import javax.ws.rs.core.HttpHeaders;
+import javax.ws.rs.core.MediaType;
+import org.broadinstitute.consent.http.configurations.ServicesConfiguration;
+import org.broadinstitute.consent.http.db.SamDAO;
+import org.broadinstitute.consent.http.models.AuthUser;
+import org.broadinstitute.consent.http.models.sam.ResourceType;
+import org.broadinstitute.consent.http.models.sam.TosResponse;
+import org.broadinstitute.consent.http.models.sam.UserStatus;
+import org.broadinstitute.consent.http.models.sam.UserStatus.Enabled;
+import org.broadinstitute.consent.http.models.sam.UserStatus.UserInfo;
+import org.broadinstitute.consent.http.models.sam.UserStatusDiagnostics;
+import org.broadinstitute.consent.http.models.sam.UserStatusInfo;
+import org.broadinstitute.consent.pact.PactTests;
+import org.junit.BeforeClass;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.experimental.categories.Category;
+
+/**
+ * Pact Consumer Contract and test for interactions between Consent and Sam
+ */
+@Category(PactTests.class)
+public class SamPactTests {
+
+  private static final String PROVIDER_NAME = "sam-provider";
+  private static final String CONSUMER_NAME = "consent-consumer";
+
+  private static final List<ResourceType> RESOURCE_TYPES = List.of(
+      new ResourceType()
+          .setName("resource-name")
+  );
+  private static final UserStatusInfo USER_STATUS_INFO =
+      new UserStatusInfo()
+          .setAdminEnabled(true)
+          .setEnabled(true)
+          .setUserEmail("test.user@gmail.com")
+          .setUserSubjectId("1234567890");
+
+  private static final UserStatusDiagnostics USER_STATUS_DIAGNOSTICS =
+      new UserStatusDiagnostics()
+          .setAdminEnabled(true)
+          .setEnabled(true)
+          .setInAllUsersGroup(true)
+          .setInGoogleProxyGroup(true)
+          .setTosAccepted(true);
+
+  private static final UserStatus USER_STATUS =
+      new UserStatus()
+          .setUserInfo(new UserInfo()
+            .setUserEmail(USER_STATUS_INFO.getUserEmail())
+            .setUserSubjectId(USER_STATUS_INFO.getUserSubjectId()))
+          .setEnabled( new Enabled()
+            .setAllUsersGroup(USER_STATUS_DIAGNOSTICS.getInAllUsersGroup())
+            .setLdap(true)
+            .setGoogle(USER_STATUS_DIAGNOSTICS.getInGoogleProxyGroup()));
```

Comment:
nit: not uniform spacing

Reply:
This is intentional - note that there is a new object being constructed inside `setUserInfo` and `setEnabled`